src/main/java/org/folio/service/fieldname/loader/IdentifierTypesLoader.java

```diff
@@ -0,0 +1,22 @@
+package org.folio.service.fieldname.loader;
+
+import io.vertx.core.json.JsonObject;
+import org.folio.clients.InventoryClient;
+import org.folio.util.OkapiConnectionParams;
+
+import java.util.Map;
+
+public class IdentifierTypesLoader implements ReferenceDataLoader {
```

Comment:
Why can't we reuse the ReferenceDataProvider? It already loads all the required reference data anyway

Reply:
We are essentially duplicating the same calls here , and will end up creating 10 more classes for the same

Reply:
I discussed it with Illia, and we thought to create a ticket for adding cach for this and do this work in the scope of that ticket

Reply:
Right even adding cache is a duplicate right, that class is already doing what we need

Reply:
Is there a special reason you need  separate classes?

Reply:
I will change it